Documentation/network/kubernetes/ciliumendpointslice_beta.rst

```diff
@@ -29,6 +29,26 @@ propagation should be reduced in this case, allowing for better scalability,
 at the cost of potentially longer delay before identity of new endpoint is
 recognized throughout the cluster.
 
+.. note::
+
+   CiliumEndpointSlice is a concept that is specific to Cilium, and not related
+   to `Kubernetes' EndpointSlice`_. Although the names are close, and even if
+   the concept of slices in each feature brings similar improvements for
+   scalability, they address different problems.
+
+   Kubernetes' Endpoints and EndpointSlices allow Cilium to make load-balancing
+   decisions for a particular Service object; EndpointSlices offer a scalable
+   way to track *Service back-ends* within a cluster.
+
+   On the other hand, CiliumEndpoints and CiliumEndpointSlices are used to make
+   network routing and policy decisions. So CiliumEndpointSlices focus on
+   tracking *Pods*, batching CEPs to reduce the number of updates to propagate
+   through the API-server on large clusters.
+
+   Enabling one does not affect the other.
+
+.. _Kubernetes' EndpointSlice: https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/
```

Comment:
Edits for clarity.

```suggestion
   CiliumEndpointSlice is a concept that is specific to Cilium and is not related
   to `Kubernetes' EndpointSlice`_. Although the names are similar, and even though
   the concept of slices in each feature brings similar improvements for
   scalability, they address different problems.

   Kubernetes' Endpoints and EndpointSlices allow Cilium to make load balancing
   decisions for a particular Service object; Kubernetes' EndpointSlices offer a scalable
   way to track *Service back-ends* within a cluster.

   By contrast, CiliumEndpoints and CiliumEndpointSlices are used to make
   network routing and policy decisions. CiliumEndpointSlices focus on
   tracking Pods, batching CEPs to reduce the number of updates to propagate
   through the API-server on large clusters.

   Enabling one does not affect the other.

.. _Kubernetes' EndpointSlice: https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/
```

Reply:
OK thanks! I always struggle to know what's best on hyphen usage on words such as “load balanc*” or “back-end”. Looks like opinions are mixed anyway, with some preference for attaching all:


```console
$ rg -Iic 'load-balanc' Documentation | awk '{a=a+$1} END {print a}' 
75
$ rg -Iic 'load balanc' Documentation | awk '{a=a+$1} END {print a}'
72
$ rg -Iic 'loadbalanc' Documentation | awk '{a=a+$1} END {print a}' 
181
```

but I prefer splitting so I'll follow your suggestion.

Reply:
@qmonnet Guidance on hyphens is mixed; I agree that it's confusing! 

In this case, removing the hyphen from "load-balancing decisions" makes it clearer that the decisions are _about_ load balancing, rather than the act of deciding somehow serving a load balancing function. Does that make sense?

Reply:
Yes it does, thanks!